lib/ansible/modules/cloud/azure/_azure.py

```diff
@@ -167,67 +168,6 @@
 from ansible.module_utils.six.moves.urllib.parse import urlparse
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.facts.timeout import TimeoutError
-
-AZURE_LOCATIONS = ['South Central US',
```

Comment:
if you remove those, it's better to add a link to supported value, so that user could have way to know what value to set

Reply:
btw, why remove those?

Reply:
these are quite old and obsolete. well, the module is quite old anyway, and i think nobody is using it anymore. i just made fixes to have all sanity tests passed.